cdap-docs/integrations/source/partners/cloudera/configuring.rst

```diff
@@ -186,10 +186,62 @@ The following is the generic procedure for Major/Minor version upgrades:
 
 .. _cloudera-release-specific-upgrade-notes:
 
+.. rubric:: Upgrading CDAP 3.0 to 3.1 and Upgrading CDH 5.3 to 5.4
+
+**Background:** CDH 5.3 ships with Hbase 0.98 while CDH 5.4 ships with HBase 1.0. We support
+CDH 5.4 as of CDAP 3.1.0. Upgrading from CDH 5.3 to CDH 5.4 entails an HBase upgrade in
+addition to a CDAP upgrade. **It is important to perform these steps as described, otherwise
+you can end up with an unusable system.**
+
+**Upgrade Steps**
+
+1. If using Cloudera Manager, :ref:`stop all CDAP application and services
+   <install-upgrade>`, as Cloudera Manager will have auto-started CDAP
+#. Disable all CDAP tables; from an HBase shell, run this command::
+
+    > disable_all 'cdap.*'
+    
+#. Upgrade to CDH 5.4
+#. :ref:`Stop CDAP application and services <install-upgrade>`, as CDH will have auto-started CDAP
+#. Upgrade to CDAP 3.1
+#. Run the CDAP Upgrade Tool, as the user that runs CDAP Master (the CDAP user)
```

Comment:
there is a way to run it from Cloudera manager. we should document them appropriately. @dereklwood can you add the instructions ?